cpg_workflows/metamist.py

```diff
@@ -197,6 +232,24 @@ def parse(data: dict) -> 'Analysis':
         return a
 
 
+def sort_sgs_by_project(response_data) -> dict:
+    """
+    Create dictionary organising sequencing groups by project
+    {project_id: [sequencing_group_1, sequencing_group_2, ...]}
+    """
+    result_dict: dict[str, Any] = {}
```

Comment:
Is this always going to be `dict[str, list[str]]` or could the dict values really be other kinds of `Any`?

Reply:
Yeah good point, this is very much a sign of me being lazy